checker/dune

```diff
@@ -1,7 +1,25 @@
-(rule (copy %{project_root}/kernel/names.ml names.ml))
-(rule (copy %{project_root}/kernel/names.mli names.mli))
-(rule (copy %{project_root}/kernel/esubst.ml esubst.ml))
-(rule (copy %{project_root}/kernel/esubst.mli esubst.mli))
+(copy_files
```

Comment:
Isn't `copy_files#` usually preferable? Cf. https://dune.readthedocs.io/en/latest/dune-files.html#copy-files

Reply:
Good question, this is a bit of a special case but indeed `copy_files#` seems to be better.

Reply:
I changed to `copy_files#`, thanks.